src/com/rcv/Main.java

```diff
@@ -164,31 +163,84 @@ static String executeTabulation(ElectionConfig config) {
   // function: parseCastVoteRecords
   // purpose: parse CVR files referenced in the ElectionConfig object into a list of CastVoteRecords
   // param: config object containing CVR file paths to parse
-  // returns: list of all CastVoteRecord objects parsed from CVR files
-  private static List<CastVoteRecord> parseCastVoteRecords(ElectionConfig config) throws Exception {
+  // returns: list of all CastVoteRecord objects parsed from CVR files (or null if there's an error)
+  private static List<CastVoteRecord> parseCastVoteRecords(ElectionConfig config) {
```

Comment:
Are there not other exceptions that could happen that aren't accounted for below? Everything works fine when we remove the `throw`?

Reply:
There's probably other stuff that can go wrong in the parsing. I think I'll test that more in a follow-up.